pyuvsim/__init__.py

```diff
@@ -18,8 +18,8 @@
 except (LookupError, ImportError):
     try:
         # Set the version automatically from the package details.
-        __version__ = get_distribution(__name__).version
-    except DistributionNotFound:  # pragma: nocover
+        __version__ = version("pyuvsim")
```

Comment:
Any concerns about this codecov warning?

Reply:
The problem here is with the structure of the package and the fact that I removed the pragma that was preventing this line from being counted. We're talking in the wider RASG community about package structure, so for now I'd like to leave this alone.